src/app/core/auth/auth.effects.ts

```diff
@@ -9,9 +9,9 @@ import { LocalStorageService } from '../local-storage/local-storage.service';
 import {
   ActionAuthLogin,
   ActionAuthLogout,
-  AUTH_KEY,
   AuthActionTypes
-} from './auth.reducer';
+} from './auth.actions';
+import { AUTH_KEY } from './auth.reducer';
```

Comment:
Let's movie `AUTH_KEY` to this file so that import is not necessary ( was done similarly for todos, stocks, ... )